launch/pipeline/service.py

```diff
@@ -1,18 +1,38 @@
 import inspect
+import uuid
 from typing import Any, Callable, Dict, List, Optional
 
 from launch.pipeline.deployment import Deployment
 from launch.pipeline.runtime import Runtime
 
+MAX_SERVICE_NAME_SIZE = 50
+
 
 class ServiceDescription:
     """
     Base ServiceDescription.
     """
 
+    service_name: str
+    service: Callable
+    runtime: Runtime
+    deployment: Deployment
+    env_params: Dict[str, str]
+
+    def __init__(self):
+        self.make_request: Optional[Callable] = None
```

Comment:
IIUC the value of `self.make_request` is either None (in which case we run the local code), or some Callable passed in from the server that makes the celery(/http?) request. Could we document this behavior? (to help people understand how the code fits together)

Reply:
I think even calling this `self.make_request_fn` would be a slight hint.